tests/test_show.py

```diff
@@ -1,14 +1,9 @@
 """Show related unit tests"""
 
-import os
-import sys
 import unittest
 
-# Load the module locally from the dev environment.
```

Comment:
This was inspired by _The Hitchhiker’s Guide to Python_ [Structuring Your Project - Test Suite](http://docs.python-guide.org/en/latest/writing/structure/#test-suite). Without the explicit `sys.path`, how do you make sure that the module is always loaded from the local development environment rather than from a possible site-wide installation? Or is `tox` taking care of this?

Reply:
AFAIK pytest takes care of this, feel free to use a very complicated sh incantation that fixes PYTHONPATH instead of just running `make test`.

Reply:
I get `error: invalid command 'pytest'` while running `make test`. Any idea what I've missed?

Reply:
Fixed in 52e47644285cb06653a8fdc11f77474f741f6796 (see commit message for details).

Reply:
This is working now, thanks.